### PR TITLE
Integration testing

### PR DESCRIPTION
**Pull request by lthiery**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -49,3 +63,67 @@ jobs:
...
+      image: ghcr.io/ledgerhq/speculos:latest
+      options: --entrypoint /bin/bash
+
+    steps:
+      - name: Clone
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          apt-get update -q
```
Comment (lthiery) [inline .github/workflows/ci.yml:99]:
```suggestion
          pip install -r tests/requirements.txt
          apt-get update -q
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -49,3 +63,68 @@ jobs:
...
+      image: ghcr.io/ledgerhq/speculos:latest
+      options: --entrypoint /bin/bash
+
+    steps:
+      - name: Clone
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          pip install -r tests/requirements.txt
```
Comment (lthiery) [inline .github/workflows/ci.yml:99]:
```suggestion
          pip install PyQt5
```

**Comment by lthiery** [comment]
@abhay still testing the CI which needs master to trigger

**Comment by abhay** [comment]
@lthiery needs to be reopened. there are some test failures worth looking at.

**Pull request merged**